src/include/common/perf_monitor.h

```diff
@@ -61,8 +61,12 @@ class PerfMonitor {
     // uint64_t branch_misses_;
     /**
      * Bus cycles, which can be different from total cycles.
+     *
+     * TODO(wz2): Dated Nov 5th, 2020 on dev4. Recording {cycle,instr,cache-ref,cache-miss,bus,ref-cpu} causes all
```

Comment:
I disabled `bus_cycles_` since adding `bus_cycles` would cause all counters to be zeroed. (this is already `kernel.perf_event_paranoid=-1` and `nmi_watchdog=0`). I disabled recording bus_cycles since we currently don't use this in the models.

There is also a new ebpf metrics collection framework under-way, so this `perf_monitor.h` might end up getting replaced with a more robust solution.

@mbutrovich